templates/suspenders_layout.html.erb.erb

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8" />
   <meta name="ROBOTS" content="NOODP" />
   <meta name="viewport" content="initial-scale=1" />
+  <!-- change the title in config/locales/en.yml in the en.titles key -->
```

Comment:
> not use the title gem at all if it's confusing to use

For what it's worth, I have never customized the title. It sets the title to "Application Name", which is fine, but anything beyond that requires research and so I don't do it.

I believe we should remove the title gem.